.github/workflows/ci.yml

```diff
@@ -210,7 +210,7 @@ jobs:
           cd ../../Microsoft/hcsshim
           git fetch --tags origin "${SHIM_COMMIT}"
           git checkout "${SHIM_COMMIT}"
-          GO111MODULE=on go build -mod=vendor -o "${bindir}/containerd-shim-runhcs-v1.exe" ./cmd/containerd-shim-runhcs-v1
+          GO111MODULE=on go build -mod=vendor -o "${bindir}/integration/client/containerd-shim-runhcs-v1.exe" ./cmd/containerd-shim-runhcs-v1
```

Comment:
Would it be better to build it into `${{ github.workspace }}/src/github.com/containerd/containerd/bin` so it's found by `PATH`, rather than relying on it being in `CWD` at the time of test execution?

Reply:
@TBBle i don't see anywhere in this action where that directory is in PATH :(

We can always follow up in another PR. Trying to do the minimum here to get it working. this adjustment was needed because the make file `cd`'s to this `integration/client` directory and containerd complains bitterly if `containerd-shim-runhcs-v1.exe` is not in the current directory.